openshift/api/values.yaml

```diff
@@ -2,6 +2,10 @@ app:
   name: met-api
   licenseplate: "e903c2"
   allowIngressPodselectors: []
+  ratelimits:
+    enabled: true
+    http: 60
```

Comment:
Might want to leave some kind of comment here that includes the unit of measurement. I see below that it's requests per 3 seconds.